packages/svelte-docgen/src/parser/events.test.ts

```diff
@@ -319,7 +319,277 @@ describe("events", () => {
 			    },
 			  },
 			  "on:increment" => {
-			    "constructors": "self",
+			    "constructors": [
```

Comment:
Surprise, surprise. I solved other bug by accident. I've overlooked this one before. I'm going to improve this test as well.